<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageLab</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_src">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>原始图像</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_dst">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>目标图像</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_test1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>450</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>测试按钮1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_test2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>450</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>测试按钮2</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="pt_log">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>440</y>
      <width>521</width>
      <height>51</height>
     </rect>
    </property>
    <property name="plainText">
     <string>测试输出
</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>编辑</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
